CMakeLists.txt

```diff
@@ -13,9 +13,31 @@ if (${CMAKE_SOURCE_DIR} STREQUAL ${CMAKE_BINARY_DIR})
     message(FATAL_ERROR "In-source builds not allowed. Please make a new directory (called a build directory) and run CMake from there. You may need to remove CMakeCache.txt.")
 endif()
 
-# determine the build type
+# determine the build type
 lug_set_option(CMAKE_BUILD_TYPE Release STRING "Choose the type of build (Debug or Release)")
 
+if(ANDROID)
+    set(ANDROID_ABI arm64-v8a)
+    message(STATUS "ANDROID_ABI : ${ANDROID_ABI}")
+    set(ANDROID_TOOLCHAIN clang)
+    message(STATUS "ANDROID_TOOLCHAIN : ${ANDROID_TOOLCHAIN} ")
+    lug_set_option(ANDROID_PLATFORM android-24 STRING "Target specific android version target (default : android-24)")
+    if(NOT ANDROID_PLATFORM MATCHES android-2[3-4])
```

Comment:
We have a full year ahead of us to even starting thinking to handle that.
I can modify for [2-9] if you want to have 5 more years planned